refs.c

```diff
@@ -1629,8 +1629,8 @@ int for_each_namespaced_ref(each_ref_fn fn, void *cb_data)
 
 int refs_for_each_rawref(struct ref_store *refs, each_ref_fn fn, void *cb_data)
 {
-	return do_for_each_ref(refs, "", fn, 0,
-			       DO_FOR_EACH_INCLUDE_BROKEN, cb_data);
+	return do_for_each_ref(refs, "refs/", fn, 0, DO_FOR_EACH_INCLUDE_BROKEN,
```

Comment:
If I were you, I would avoid pulling the `DO_FOR_EACH_INCLUDE_BROKEN` flag into the previous line: this is a change you do not need to make, and it is distracting from the _actual_ change (on the Git mailing list, you don't have color, so it will be even harder to review what you did).

Reply:
this is clang-format's doing.  I prefer to let the tools make mechanical decisions; they're much better at it than I am.